docs/issues/58ee357bfd50147615170c189a8a14c6.md

```
@@ -12,14 +12,32 @@ Severity: medium
 [View in Firebase Console](https://console.firebase.google.com/project/d-print-cost-calculator-cf650/crashlytics/app/android:com.threed_print_calculator/issues/58ee357bfd50147615170c189a8a14c6)
 
 ## Summary
-ANR triggered by main thread waiting for too long
+Main thread ANR occurring during the handling of an uncaught exception by the Firebase Crashlytics SDK.
 
 ## Summary
 ANR triggered by main thread waiting for CountDownLatch.await (binder transaction to system server)
 
-## Suspected cause
+## Stack Trace
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)